docs/user/package-management/history-and-rollback.md

```diff
@@ -3,34 +3,57 @@ title: History and Software Rollback
 summary: History and Software Rollback
 ---
 
-# History and Software Rollback
+# History and software rollback in Solus
 
-Solus provides a history and rollback feature via its package manager, `eopkg`. This feature allows you to see when your repository was last updated, actions such as installation, removal, and upgrading of software.
+Solus offers a feature to view your history of software changes and revert your system to previous versions. 
 
-Our rollback feature allows you to essentially rewind your system's software back to a previous state, assuming you or our repository has the versions in question.
+You can use this feature if you have issues after installing packages or system updates, and you need to return to a working state.
 
-## History
+:::warning[Important]
 
-You can see the history from `eopkg` by using:
+The rollback operation only works if:
 
-```bash
-eopkg history
-```
+- The Solus repository has the version of the package you need.
+- You have a local copy of the package.
+
+:::
+
+## History of software changes
+
+- To see your history of software changes, open a terminal and execute:
+
+    ```bash
+    eopkg history
+    ```
+
+    The terminal displays a list of all the software changes in your system.
+
+    ![eopkg history output](./eopkg-history-output.png)
```

Comment:
Is this meant to be a list item?

Reply:
It's the result of the [single-step procedure](https://developers.google.com/style/procedures#single-step-procedures). The extra indentation is just so it's rendered as part of the bullet point:

![image](https://github.com/user-attachments/assets/16782e75-55f0-48a5-93b9-c15509a45e46)

Instead of this:
![image](https://github.com/user-attachments/assets/6e49bc9a-e375-42e3-8fec-9f9b04f242ef)